<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeInfo</class>
 <widget class="QWidget" name="ChangeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>If you want to change your username enter 
 the new username 
 else, enter your current username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>If you want to change your password enter 
 the new password 
 else, enter your current password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changesdone">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Finished</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter your student number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="saved">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Changes have been successfuly saved</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sorry! your password must contain at least one digit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
